src/api/java/com/minecolonies/coremod/colony/buildings/IBuilding.java

```diff
@@ -0,0 +1,53 @@
+package com.minecolonies.coremod.colony.buildings;
+
+import com.minecolonies.coremod.colony.IColony;
+import net.minecraft.util.math.BlockPos;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * Created by marcf on 3/7/2017.
+ */
+public interface IBuilding {
```

Comment:
I think this belong to a different pr, and could be merge well before dman 2.0 is ready